events/artsy-learns-to-code-v2.md

```diff
@@ -0,0 +1,116 @@
+
+We'll start with this youtube tutorial, which is taught by a professor at NYU's ITP (Interactive Telecommunications
+Program):
+
+https://www.youtube.com/user/shiffman/playlists?view=50&sort=dd&shelf_id=2
+
+In an effort to keep attendance up and keep learners motivated, we won't enforce a schedule that all learners
+should be following. Instead, groups can set their own cadence and work on a schedule that works for them.
+
+Then groups can decide on what they might want to make with p5.js, and mentors can guide them.
+
+### Some Core Competencies We Could Aim Teach
+
+- Fundamentals:
+  - Syntax
+  - Data types & Variables
+  - Conditionals
+  - Functions
+  - Loops
+  - Arrays
+  - Objects
+- Web Development:
+  - What is a library/package?
+  - How does JS/CSS/HTML interact with each other?
+  - What is a server? A client?
+- More General Software Development:
+  - Algorithms
+  - Classes/OOP
+  - SQL
+
+## Expectations of Learners
+
+- Passion to learn and try something new
+- An open mind
+- (Optional) Come with an idea of what they want to learn
+
+## Expectations of Mentoring Engineers
+
+- Mentors should expect to spend 1-2 hrs a week of their time to meet regularly and answer questions async on slack
+- Mentors should feel empowered to come up with their own teaching plan (and ideally share it with the rest of the
+  mentors!). Note: mentors should consider the extra time required to craft their own teaching plan.
+- Act as a resource for when learners have questions
+- Act as a motivator for when learners get frustrated
+  - Normalize frustration (https://www.youtube.com/watch?v=QYjPBcqo1-s)
+- If mentor group agrees to go the route of learning Processing.js, then the mentor should try to familiarize
+  themselves with p5js, or at least follow along with the course.
+
+## Teaching resources
+
+Here are a couple examples that [@sepans][sepans] have created:
+
+| Project Name + Concept it teaches                | Link                                              |
+| ------------------------------------------------ | ------------------------------------------------- |
+| Bouncy bubble (conditionals)                     | https://editor.p5js.org/sepans/sketches/8I6gA3O-1 |
+| Damien Hirst (loops)                             | https://editor.p5js.org/sepans/sketches/o34yoVD3y |
+| Damien Hist 2 (array to store randomized colors) | https://editor.p5js.org/sepans/sketches/RBjEojE3g |
+| Bouncy DVD with Object (objects)                 | https://editor.p5js.org/sepans/sketches/jmu-Gb0u7 |
+| 300 DVDs (objects + functions + arrays)          | https://editor.p5js.org/sepans/sketches/yaafYKYN7 |
+| Sunglass (capturing data and displaying it)      | https://editor.p5js.org/sepans/sketches/s9tmMft3R |
+| Load artwork image from artsy (API)              | https://editor.p5js.org/sepans/sketches/NbuDfgC47 |
+
+Here are tools that [@sepans][sepans] recommended to his group in the previous ALTC:
```

Comment:
Maybe I wasn't clear about the purpose if these tools. These are the ones for those who wanted to move up and use a more general purpose editor instead of p5 editor otherwise p5 editor more or less support these. So we can move this list to the bottom of the doc only for those who want to use something other than p5 (or drop it if it is no longer relevant).